packages/core/src/external/athenahealth/index.ts

```diff
@@ -322,21 +376,93 @@ class AthenaHealthApi {
     return medicationOptions;
   }
 
+  async getAppointments({
+    cxId,
+    departmentId,
+    startAppointmentDate,
+    endAppointmentDate,
+    startLastModifiedDate,
+    endLastModifiedDate,
+  }: {
+    cxId: string;
+    departmentId: string;
+    startAppointmentDate: Date;
+    endAppointmentDate: Date;
+    startLastModifiedDate: Date;
+    endLastModifiedDate: Date;
+  }): Promise<{ appointments: AthenaAppointment[] }> {
+    const { log, debug } = out(`AthenaHealth get appointments`);
```

Comment:
afaik we instantiate logs with more details (i.e. including `cxId`, `patientId`, etc. Otherwise, how are you gonna tell apart the logs from different calls

Reply:
Good catch. Done.